src/main/java/org/kainos/ea/resources/AuthController.java

```diff
@@ -0,0 +1,55 @@
+package org.kainos.ea.resources;
+
+import io.swagger.annotations.Api;
+import org.kainos.ea.api.AuthService;
+import org.kainos.ea.cli.Login;
+import org.kainos.ea.cli.Register;
+import org.kainos.ea.client.FailedToGenerateTokenException;
+import org.kainos.ea.client.FailedToLoginException;
+import org.kainos.ea.client.FailedToRegisterException;
+
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Api("Belfast_Team2 Auth API")
+@Path("/api")
+public class AuthController {
+
+    private AuthService authService = new AuthService();
+
+    @POST
+    @Path("/login")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response login(Login login) {
+        try {
+            return Response.ok(authService.login(login)).build();
+        } catch (FailedToLoginException e) {
+            System.err.println(e.getMessage());
+
+            return Response.status(Response.Status.BAD_REQUEST).entity(e.getMessage()).build();
+        } catch (FailedToGenerateTokenException e) {
+            System.err.println(e.getMessage());
+
+            return Response.status(Response.Status.INTERNAL_SERVER_ERROR).entity(e.getMessage()).build();
+        }
+    }
+
+    @POST
+    @Path("/register")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response validRegister(Register register) {
+        try {
+            authService.register(register);
+
+            return Response.ok().build();
```

Comment:
Would `return Response.ok(authService.register(register);).build();` be easier to read here